docs/UserGuide.md

```diff
@@ -32,80 +32,173 @@
 > **Command Format**
 > * Words in `UPPER_CASE` are the parameters.
 > * Items in `SQUARE_BRACKETS` are optional.
-> * Items with `...` after them can have multiple instances.
-> * The order of parameters is fixed.
+> * The order of parameters is fixed, with optional parameters being the exception.
+> * Parameters `START_DATE` and `END_DATE` has to be in dd-mm-yyyy format (e.g. 23-10-2016).
+> * Parameters `START_TIME` and `END_TIME` has to be in 24 hour format (e.g. 1215).
+> * Parameter `TIMELEFT` **must be a positive integer**, indicating number of hours left.
 
 #### Viewing help : `help`
 Format: `help`
 
 > Help is also shown if you enter an incorrect command e.g. `abcd`
  
-#### Adding a person: `add`
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` 
+#### Adding a task : `add`
+Adds a task to the task manager.<br>
+Format: `add TASK_NAME [sd/START_DATE] [st/START_TIME] [ed/END_DATE] [et/END_TIME] [tl/TIMELEFT]` 
 
-> Persons can have any number of tags (including 0)
+> `TASK_NAME` is required, but the rest of the parameters are optional.<br>
+  User can specify `END_DATE` or `TIME_LEFT` to set a deadline to a task.<br>
+  User can specify `START_DATE`, `START_TIME`, `END_DATE` and `END_TIME` to set an event to the task.
 
 Examples: 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe p/1234567 e/betsycrowe@gmail.com a/Newgate Prison t/criminal t/friend`
+* `add follow up with Jack on sales report sd/6-10-2016`
```

Comment:
hmmm.. this example doesn't follow the description above. you say that user can specify `END_DATE` to set deadline, but this example uses start date. does it mean, when only one date is involved it is always considered as deadline? (it would be cool to revert to this default behavior! )